content/en/dynamic_instrumentation/enabling/nodejs.md

```diff
@@ -0,0 +1,106 @@
+
+Dynamic Instrumentation is a feature of supporting Datadog tracing libraries. If you are already using [APM to collect traces][1] for your application, ensure your Agent and tracing library are on the required version. Then, go directly to enabling Dynamic Instrumentation in step 4.
+
+## Installation
+
+1. Install or upgrade your Agent to version [7.45.0][7] or higher.
+2. If you don't already have APM enabled, in your Agent configuration, set the `DD_APM_ENABLED` environment variable to `true` and listening to the port `8126/TCP`.
+3. Install or upgrade the Node.js tracing library to version 5.32.0 or higher, by following the [relevant instructions][2].
+4. Run your service with Dynamic Instrumentation enabled by setting the `DD_DYNAMIC_INSTRUMENTATION_ENABLED` environment variable to `true`. Specify `DD_SERVICE`, `DD_ENV`, and `DD_VERSION` Unified Service Tags so you can filter and group your instrumentations and target active clients across these dimensions.
+5. After starting your service with Dynamic Instrumentation enabled, you can start using Dynamic Instrumentation on the [APM > Dynamic Instrumentation page][3].
+
+## Configuration
+
+Configure Dynamic Instrumentation using the following environment variables:
+
+| Environment variable                             | Type          | Description                                                                                                               |
+| ------------------------------------------------ | ------------- | ------------------------------------------------------------------------------------------------------------------------- |
+| `DD_DYNAMIC_INSTRUMENTATION_ENABLED`             | Boolean       | Set to `true` to enable Dynamic Instrumentation.                                                                          |
+| `DD_SERVICE`                                     | String        | The [service][4] name, for example, `web-backend`.                                                                        |
+| `DD_ENV`                                         | String        | The [environment][4] name, for example, `production`.                                                                     |
+| `DD_VERSION`                                     | String        | The [version][4] of your service.                                                                                         |
+| `DD_TAGS`                                        | String        | Tags to apply to produced data. Must be a list of `<key>:<value>` separated by commas such as: `layer:api,team:intake`.   |
+
+## What to do next
+
+See [Dynamic Instrumentation][5] for information about adding instrumentations and browsing and indexing the data.
+
+## Limitations
+
+Dynamic Instrumentation for Node.js is currently in limited preview, which means it is not available to all customers. To join the waiting list, [fill out this form][6].
+
+The following limitations apply to the limited preview:
```

Comment:
@sstonehill12 I'm considering just dropping one or both of these paragraphs. The first paragraph is already included in the callout at the top of the page and the second doesn't really add any value. Let me know what you think.